versions.tf

```diff
@@ -30,12 +30,12 @@ terraform {
     }
     # Required by the EKS module
     helm = {
-      source = "hashicorp/helm"
```

Comment:
This change is unwanted. You might want to setup your text editor to trim empty space on end lines